base/src/main/java/com/mindera/skeletoid/webview/cookies/CookiesUtils.kt

```diff
@@ -34,28 +34,32 @@ object CookiesUtils {
         return cookiesMap
     }
 
-    fun getCookie(url: String, cookieName: String): String? {
+    fun getCookie(url: String, cookieName: String): Map<String,String> {
```

Comment:
missing space after comma

Reply:
Isn't this method just a particular case of the method above ("getCookiesMap")? From what I can see, the method above takes the cookies, parses them and returns a map with them. This method takes the cookies, parses them and then filters them. Unless there is something I'm missing couldn't we use the method above and filter the output?